### ISSUE TITLE
Add toggle to disable color codes

### ISSUE THREAD

**Issue by luciensadi** [luciensadi/AwakeMUD#119]
The ability to turn off color would be nice for when you're looking to play without the game looking so much like a game. Maybe TOGGLE [COLOR|COLOUR]. This functionality probably already exists in the screenreader code and could be cannibalized easily (if not, it should be there-- screenreaders don't benefit from color, right?)

**Comment by luciensadi** [comment]
Implemented TOGGLE NOCOLOR in screenreader_improvements.
